woocommerce/woocommerce-android
# Order List redesign - Step 2: Enhance search support to the order list UI

Fixes #1400. This PR completes the second step of the Stats Improvements Project as defined in the master thread [here](https://github.com/woocommerce/woocommerce-android/issues/1398). 

### Changes
- Added a new class `OrderStatusListView` that displays the order status list UI when search icon is selected.
- Removed the filter option from the toolbar menu.

**Note that this PR is to be merged to a feature branch.**

### Screenshots
<img width="300" src="https://user-images.githubusercontent.com/22608780/65567380-abea4180-df73-11e9-8fe4-c1b4b9273a06.gif"> . <img width="300" src="https://user-images.githubusercontent.com/22608780/65567382-adb40500-df73-11e9-9413-3f5280e7be8c.gif"> . <img width="300" src="https://user-images.githubusercontent.com/22608780/65567417-dc31e000-df73-11e9-9ca7-db85f79a68d6.png">


### Testing
- Search for an order by clicking on an order status.
- Search for an order by typing a search query.
- Verify that pagination, search, filter works as expected.

Update release notes:
